code/modules/power/apc.dm

```diff
@@ -1253,9 +1254,10 @@
 		force_update = TRUE
 		return
 
-	lastused_light = area.power_usage[AREA_USAGE_LIGHT] + area.power_usage[AREA_USAGE_STATIC_LIGHT]
-	lastused_equip = area.power_usage[AREA_USAGE_EQUIP] + area.power_usage[AREA_USAGE_STATIC_EQUIP]
-	lastused_environ = area.power_usage[AREA_USAGE_ENVIRON] + area.power_usage[AREA_USAGE_STATIC_ENVIRON]
+	//dont use any power from that channel if we shut it off
+	lastused_light = (area.power_usage[AREA_USAGE_LIGHT] + area.power_usage[AREA_USAGE_STATIC_LIGHT]) * (lighting > APC_CHANNEL_AUTO_OFF)
+	lastused_equip = (area.power_usage[AREA_USAGE_EQUIP] + area.power_usage[AREA_USAGE_STATIC_EQUIP]) * (equipment > APC_CHANNEL_AUTO_OFF)
+	lastused_environ = (area.power_usage[AREA_USAGE_ENVIRON] + area.power_usage[AREA_USAGE_STATIC_ENVIRON]) * (environ > APC_CHANNEL_AUTO_OFF)
```

Comment:
i believe this was a hack to make static power usage not draw if the apc turns that channel off which is incompatible with making 99% of things use static power. ill need to find a better solution for this i think

Reply:
actually ill have to think about this, the way that dynamic power usage is cleared is to literally set the areas power channel to 0 after recording how much that channel used but you cant do that for static power since thats the entire point of it